src/components/MessageInput/EditMessageForm.js

```diff
@@ -150,38 +144,20 @@ class EditMessageForm extends React.Component {
         <ImageDropzone
           accept={this.props.acceptedFiles}
           multiple={this.props.multipleUploads}
-          disabled={
-            this.props.numberOfUploads >= this.props.maxNumberOfFiles
-              ? true
-              : false
-          }
+          disabled={this.props.numberOfUploads >= this.props.maxNumberOfFiles}
           handleFiles={this.props.uploadNewFiles}
         >
           <form onSubmit={this.props.handleSubmit}>
             {this.renderEmojiPicker()}
             {this.renderUploads()}
-            {this.props.emojiPickerIsOpen && (
```

Comment:
does this mean we were rendering emojiPicker twice?

Reply:
Yep :)